### DISCUSSION TITLE
Programming a brain for MaSh

### DISCUSSION THREAD

**Discussion by BOSS294** [BOSS294/PROJECT-991#14] [Q&A] [answered]
Will it be good to create a brain ( DATABASE ) for MaSh where he stores each and every information and tasks 

Like if we feed him about what he is , he will automatically store it into his brain ( DATABASE ) 

and work accordingly, I know this is complicated but is their any simpler way of doing it ??

( USING "HE" becuase i term MaSh as our BOY )

**Comment by jindalpriyanshu101** [comment, marked as answer]
The same thing was answered earlier in your question about future of MaSh, However for your reference. We will be utilising NLP and ML Cases for such thing, where `meaningful tokens` (prompt that generated an output) will be stored together and `MaSh's` algorithm will be designed to utilise those tokens first before generating any new response to save our resources during OpenAI API integration.

After this implementation, significant increase in MaSh's response and efficiency to a prompt is expected. Let's hope for the best :)
